src/Sarif/ExtensionMethods.cs

```diff
@@ -145,6 +145,11 @@ public static bool IsEqualToOrHierarchicalDescendantOf(this string child, string
             return true;
         }
 
+        public static string UseLinuxNewline(this string input)
```

Comment:
>UseLinuxNewline [](http://example.com/codeflow?start=29&length=15)

Let's call this 'NormalizeToLinuxNewlines'.

Reply:
updated